public/src/components/channelManagement/landingPageTierEditor.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles(({ spacing }: Theme) => ({
+  container: {
+    '& > * + *': {
+      marginTop: spacing(1),
+    },
+  },
+  fieldsContainer: {
+    '& > * + *': {
+      marginTop: spacing(3),
+    },
+  },
+}));
+
+interface FormData {}
+
+interface LandingPageTierEditorProps {}
+
+const LandingPageTierEditor: React.FC<LandingPageTierEditorProps> = ({}: LandingPageTierEditorProps) => {
```

Comment:
this seems to be unused

Reply:
oh yes, good catch , I will remove that.Also just now it strikes to me that we need to remove the preview (live/web) options as well